ocs_ci/ocs/resources/backingstore.py

```
@@ -165,6 +165,22 @@ def _cli_deletion_flow():
+                    f"Failed to delete backingstore {self.name} after {timeout} seconds"
+                )
+                try:
+                    log.info(f"Getting describe output for backingstore {self.name}")
+                    bs_describe = OCP(
+                        kind=constants.BACKINGSTORE,
+                        namespace=config.ENV_DATA["cluster_namespace"],
+                    ).describe(resource_name=self.name)
+                    log.error(
+                        f"Backingstore {self.name} describe output:\n{bs_describe}"
+                    )
+                except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)